packages/vats/test/upgrading/test-upgrade-vats.js

```diff
@@ -17,6 +17,11 @@ const makeTestContext = async metaUrl => {
   return { bfile };
 };
 
+const makeCallOutbound = t => (srcID, obj) => {
```

Comment:
we end up doing this a lot to provide `t`. I wish we had some handy curry functions